conflowgen/previews/modal_split_preview.py

```diff
@@ -68,7 +68,8 @@ def hypothesize_with_mode_of_transport_distribution(
 
     def get_transshipment_and_hinterland_share(self) -> TransshipmentAndHinterlandComparison:
         """
-        Returns: The amount of containers in TEU dedicated for or coming from the hinterland versus the amount of
+        Returns:
+             The amount of containers in TEU dedicated for or coming from the hinterland versus the amount of
```

Comment:
This looks broken

![grafik](https://user-images.githubusercontent.com/5236165/152528783-e914a6e3-8f28-49af-88f3-f2b42130f581.png)